app/controllers/restaurants_controller.rb

```diff
@@ -2,6 +2,7 @@ class RestaurantsController < ApplicationController
   before_action :authorize_user, except: [:index, :show, :create, :new]
 
   def index
+    @google_maps_url = 'https://www.google.com/maps/embed/v1/place?key=' + ENV["GOOGLE_MAPS_API_KEY"] + '&q=Chinatown+Boston+MA'
```

Comment:
Line is too long. [128/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.